### PR TITLE
Add testcases for libssh transport

### PR DESCRIPTION
**Pull request by ganeshrn**
Depends-On: https://github.com/ansible-collections/ansible.netcommon/pull/30
##### SUMMARY

*  Add test cases with ansible_network_cli_ssh_type set to libssh
   to test libssh connection plugin

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

**Comment by ganeshrn** [comment]
recheck

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ansible
```
Comment (webknjaz) [inline requirements.txt:1]:
```suggestion
-i https://test.pypi.org/simple/

ansible
```

Comment (ganeshrn) [reply requirements.txt:2]: Fixed

**Comment by ganeshrn** [comment]
recheck

File: tests/integration/targets/ios_acl_interfaces/tasks/cli.yaml [hunk 2]
```diff
@@ -11,10 +11,19 @@
   set_fact: test_items="{{ test_cases.files | map(attribute='path') | list }}"
   delegate_to: localhost
 
-- name: Run test case (connection=ansible.netcommon.network_cli)
+- name: Run test case (connection=ansible.netcommon.network_cli with paramiko transport)
   include: '{{ test_case_to_run }}'
   vars:
     ansible_connection: ansible.netcommon.network_cli
```
Comment (Qalthos) [inline tests/integration/targets/ios_acl_interfaces/tasks/cli.yaml:17]:
```suggestion
    ansible_connection: ansible.netcommon.network_cli
    ansible_network_cli_ssh_type: paramiko
```

It's unnecessary, but would be more explicit about what we intend to test. And if we change the default down the line, continues to do what we intend

File: tests/integration/targets/ios_acl_interfaces/tasks/cli.yaml [hunk 3]
```diff
@@ -11,10 +11,19 @@
...
   with_items: '{{ test_items }}'
   loop_control:
     loop_var: test_case_to_run
```
Comment (Qalthos) [inline tests/integration/targets/ios_acl_interfaces/tasks/cli.yaml:20]:
```suggestion
    loop_var: test_case_to_run
  tags: paramiko
```

File: tests/integration/targets/ios_acl_interfaces/tasks/cli.yaml [hunk 4]
```diff
@@ -11,10 +11,19 @@
...
+
+- name: Run test case (connection=ansible.netcommon.network_cli with libssh transport)
+  include: '{{ test_case_to_run }}'
+  vars:
+    ansible_connection: ansible.netcommon.network_cli
+    ansible_network_cli_ssh_type: libssh
+  with_items: '{{ test_items }}'
+  loop_control:
+    loop_var: test_case_to_run
```
Comment (Qalthos) [inline tests/integration/targets/ios_acl_interfaces/tasks/cli.yaml:29]:
```suggestion
    loop_var: test_case_to_run
  tags: libssh
```

**Comment by Andersson007** [comment]
recheck

**Comment by Andersson007** [comment]
@Qalthos @ganeshrn @webknjaz can we merge this?

**Comment by webknjaz** [comment]
I don't have access to this repo

**Comment by ganeshrn** [comment]
The testing for libssh is added as part of zuul job hence this PR is no longer required

**Comment by Andersson007** [comment]
thank you!

**Pull request closed without merging**